INSTALL.md

```diff
@@ -0,0 +1,140 @@
+started.
+
+```console
+$ pip3 install docker-compose
+```
+
+If you are not using virtualenv,
+
+```console
+$ sudo pip install docker-compose
+```
+
+> pip version 6.0 or greater is required.
+
+#### Install as a container
+
+Compose can also be run inside a container, from a small bash script wrapper. To
+install compose as a container run this command:
+
+```console
+$ sudo curl -L --fail https://github.com/docker/compose/releases/download/1.29.2/run.sh -o /usr/local/bin/docker-compose
+$ sudo chmod +x /usr/local/bin/docker-compose
+```
+
+
+### Install pre-release builds
+
+If you're interested in trying out a pre-release build, you can download release
+candidates from the [Compose repository release page on GitHub](https://github.com/docker/compose/releases).
+Follow the instructions from the link, which involves running the `curl` command
+in your terminal to download the binaries.
+
+Pre-releases built from the "master" branch are also available for download at
+[https://dl.bintray.com/docker-compose/master/](https://dl.bintray.com/docker-compose/master/).
+
+> Pre-release builds allow you to try out new features before they are released,
+> but may be less stable.
+
+----
+
+## Upgrading
+
+If you're upgrading from Compose 1.2 or earlier, remove or
+migrate your existing containers after upgrading Compose. This is because, as of
+version 1.3, Compose uses Docker labels to keep track of containers, and your
+containers need to be recreated to add the labels.
+
+If Compose detects containers that were created without labels, it refuses
+to run, so that you don't end up with two sets of them. If you want to keep using
+your existing containers (for example, because they have data volumes you want
+to preserve), you can use Compose 1.5.x to migrate them with the following
+command:
+
+```console
+$ docker-compose migrate-to-labels
+```
+
+Alternatively, if you're not worried about keeping them, you can remove them.
+Compose just creates new ones.
+
+```console
+$ docker container rm -f -v myapp_web_1 myapp_db_1 ...
+```
+
+## Uninstallation
+
+To uninstall Docker Compose if you installed using `curl`:
+
+```console
+$ sudo rm /usr/local/bin/docker-compose
+```
+
+To uninstall Docker Compose if you installed using `pip`:
+
+```console
+$ pip uninstall docker-compose
+```
+
+> Got a "Permission denied" error?
+>
+> If you get a "Permission denied" error using either of the above
+> methods, you probably do not have the proper permissions to remove
+> `docker-compose`. To force the removal, prepend `sudo` to either of the above
+> commands and run again.
+
+## Using Compose v1 and Compose v2 on the same computer
+
+If you need to run both `v1` and `v2` versions of Compose you can use Compose Switch project that will allow you to 
+choose which version of compose to run.
+
+You can find all the installation and configuration step in the [project repository](https://github.com/docker/compose-switch)
```

Comment:
do we really want to promote compose-switch? compose v2 can be used standalone, so v1 vs v2 is just a question on setting `PATH` to select the right one

Reply:
removed 👍